server/usecases/get-lsc-status.lua

```diff
@@ -30,8 +30,8 @@ local function exec(address, name)
     if problems > 0 then state = states.BROKEN end
     
     local status = {
-        storedEU = lsc:getStoredEU(),
-        EUCapacity = lsc:getEUCapacity(),
+        storedEU = math.floor(string.gsub(lsc.getSensorInformation()[2], "([^0-9]+)", "")),
```

Comment:
There's a specific usecase for that
Use the same `sensorInformation` for both if the rate is limited

Reply:
Yeah I noticed that after the commit
It's fixed in a further commit.